account_invoice_line_report/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Binh Lam <binh.lt@komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Account Invoice Line Report',
+    'summary': 'New views to manage invoice lines information',
+    'version': '10.0.1.0.0',
+    'category': 'Account',
+    'author': 'Binh Lam, '
```

Comment:
You should add the original author: `Tecnativa`

you should add yourself aslo.